test/Cruzo1155Vault.ts

```diff
@@ -0,0 +1,300 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { ethers, upgrades } from "hardhat";
+import { Contract } from "ethers";
+import { getEvent } from "../utils/getEvent";
+import { keccak256 } from "ethers/lib/utils";
+import { RAW_VAULT_FUNCTION_SIGNATURE, RAW_FACTORY_INITIALIZE_SIGNATURE } from "../constants/signatures"
+
+describe("CruzoVault", () => {
+    let market: Contract;
+    let beacon: Contract;
+    let factory: Contract;
+    let token: Contract;
+    let vault: Contract;
+    let tokenV2: Contract;
+    let owner: SignerWithAddress;
+    let seller: SignerWithAddress;
+    let buyer: SignerWithAddress;
+    let claimer: SignerWithAddress;
+    let claimer2: SignerWithAddress;
+
+
+    const serviceFee = 300;
+    const rawVaultFuncSignature = RAW_VAULT_FUNCTION_SIGNATURE
```

Comment:
Meaningless assignment, you can just use the imported RAW_VAULT_FUNCTION_SIGNATURE value directly.